gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixTask.java

```diff
@@ -213,9 +213,17 @@ private Integer getPartitionForHelixTask(TaskDriver taskDriver) {
 
   @Override
   public void cancel() {
-    log.info("Gobblin helix task cancellation invoked.");
+    log.info("Gobblin helix task cancellation invoked for jobId {}.", jobId);
     if (this.task != null ) {
-      this.task.cancel();
+      try {
+        this.task.cancel();
+        log.info("Gobblin helix task cancellation completed for jobId {}.", jobId);
+      } catch (Throwable t) {
+        log.info("Gobblin helix task cancellation for jobId {} failed with exception.", jobId, t);
```

Comment:
I don't think this log line is needed, given that the RuntimeException already has the context. This will pollute the logs with duplicate stacktraces.

Reply:
I do not intend to just catch and throw the Throwable. I wanted to log the exception in gobblin's log. This method is called from helix class and I do not want to depend on helix code on exception handling.